src/transactions/typings/txAttrUsage.d.ts

```diff
@@ -0,0 +1,41 @@
+/** Enum for TransactionAttributeUsage */
+export interface TxAttrUsage {
```

Comment:
Not an expert on typings but I recall Typescript having enums. Is `interface` correct here?

Reply:
updated to enum along with sc.opCode